udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,9 +25,25 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> comments;
+
+  /**
+   * Converts a ServerStats instance into a JSON string using the Gson library. Note: We first added
+   * the Gson library dependency to pom.xml.
+   */
+  private String arrayListToJson(ArrayList<String> inputList) {
```

Comment:
Consider making the input argument more generic. Since `toJson()` accepts List, this method could also accept List (instead of ArrayList), to make the method more reusable.

Reply:
Will update in step 6.